lua/entities/gmod_wire_expression2/core/custom/vgui_elements/server/dtextentry.lua

```diff
@@ -150,3 +150,17 @@ do--[[getter]]--
     end
 -- getter
 end
+
+do--[[setter]]--
```

Comment:
We should either group the setters and getters or remove this do...end block completly since it serves no real purpose other than code folding (which is only needed in some files with more methods)

Reply:
Yeah, alright. Was just following the convention in the file. I'll remove the do blocks.